doc/nrf/releases_and_maturity/releases/release-notes-3.1.0.rst

```diff
@@ -0,0 +1,1320 @@
+
+* :ref:`matter_light_switch_sample`:
+
+  * Updated the testing steps to use the proper commands for groupcast binding.
+    This resolves the :ref:`known issue <known_issues>` KRKNWK-19277.
+
+Networking samples
+------------------
+
+* Added support for the nRF7002 EK with the ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples:
+
+  * :ref:`download_sample`
+  * :ref:`net_coap_client_sample`
+  * :ref:`https_client`
+  * :ref:`mqtt_sample`
+  * :ref:`udp_sample`
+  * :ref:`http_server`
+
+* :ref:`download_sample` sample:
+
+  * Added the :ref:`CONFIG_SAMPLE_PROVISION_CERT <CONFIG_SAMPLE_PROVISION_CERT>` Kconfig option to provision the root CA certificate to the modem.
+    The certificate is provisioned only if the :ref:`CONFIG_SAMPLE_SECURE_SOCKET <CONFIG_SAMPLE_SECURE_SOCKET>` Kconfig option is set to ``y``.
+  * Fixed an issue where the network interface was not re-initialized after a fault.
+
+NFC samples
+-----------
+
+* Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` board target to the following samples:
+
+  * :ref:`writable_ndef_msg`
+  * :ref:`nfc_shell`
+
+* :ref:`record_text` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
+
+nRF5340 samples
+---------------
+
+|no_changes_yet_note|
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` board target.
+
+* :ref:`802154_phy_test` sample:
+
+  * Added print of sent packets and received Acks after ``ltx`` command.
+
+PMIC samples
+------------
+
+* :ref:`npm1300_fuel_gauge` sample:
+
+  * Updated current sign when calling the ``nrf_fuel_gauge`` lib functions.
+    In the Zephyr sensor API the convention for the gauge current is ``negative=discharging``, while the :ref:`nrfxlib:nrf_fuel_gauge` lib uses the opposite.
+
+Protocol serialization samples
+------------------------------
+
+|no_changes_yet_note|
+
+SDFW samples
+------------
+
+|no_changes_yet_note|
+
+Sensor samples
+--------------
+
+|no_changes_yet_note|
+
+SUIT samples
+------------
+
+* Removed all SUIT samples:
+
+  * SUIT: Device firmware “A/B” update on the nRF54H20 SoC
+  * SUIT: Flash companion
+  * SUIT: Recovery application
+  * SUIT: Device firmware update on the nRF54H20 SoC
+
+Trusted Firmware-M (TF-M) samples
+---------------------------------
+
+* :ref:`tfm_secure_peripheral_partition` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
+
+Thread samples
+--------------
+
+* Added the new :ref:`architecture option <openthread_stack_architecture>` to use the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver in the following samples:
+
+  * :ref:`ot_coprocessor_sample`
+  * :ref:`coap_server_sample`
+  * :ref:`ot_cli_sample`
+
+Wi-Fi samples
+-------------
+
+* Added runtime support for Buletooth LE based Wi-Fi provisioning in enterprise mode.
```

Comment:
Please add under the Added section for https://github.com/nrfconnect/sdk-nrf/pull/23552 PR, (@krish2718 to confirm) :

* The :ref:`wifi_provisioning_internal_sample` sample that demonstrates the core Wi-Fi provisioning functionality independently of the transport layer (Bluetooth LE or Wi-Fi SoftAP).